scripts/jest.config.js

```diff
@@ -0,0 +1,20 @@
+import { pathsToModuleNameMapper } from 'ts-jest'
+import tsconfig from './tsconfig.json' assert { type: 'json' }
```

Comment:
Since we are using import assertions, we should probably bump the TypeScript version in `package.json` to `^5.3.0`.